### PR TITLE
Create expense-reimbursements.md

### PR DESCRIPTION
**Pull request by monicazng**

### CODE REVIEW

File: src/finance/expense-reimbursements.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Expense reimbursements
+
+![alt text](https://images.unsplash.com/photo-1521540216272-a50305cd4421?ixlib=rb-0.3.5&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=1080&fit=max&ixid=eyJhcHBfaWQiOjExNzczfQ&s=611185f92103f088fb8dfb682f810e19)
```
Comment (johannesmw) [inline src/finance/expense-reimbursements.md:3]: We could probably take this image out?

File: src/finance/expense-reimbursements.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+![expensify-expense-details](/src/images/expensify-expense-details.png/)
+
+   3. If it's your first expense of the month, create a new report for all costs incurred during that period. Otherwise, add it to an existing one.
+   4. We suggest submitting an expense reimbursement report only when the total value is above USD 50.
+   5. You **must** submit expenses as expenses rather than an invoice.
+
+![expensify-expenses-expenses](/src/images/expensify-expenses-expenses.png)
+
+> ⚠️ You **must always** need to **submit a receipt with each expense** you log. Make sure to take photos or keep digital receipts handy - missing receipts might result in your reimbursements not being approved. We understand that in some rare cases, this is not possible. When not possible, please provide as much info as possible and why no receipt is possible.
```
Comment (johannesmw) [inline src/finance/expense-reimbursements.md:55]: You **must always** need to **submit a receipt with each expense** you log. 

Change to:
You **always need to submit a receipt with each expense** you log.

File: src/finance/expense-reimbursements.md (hunk 1 above)
Comment (monicazng) [reply src/finance/expense-reimbursements.md:3]: +1

File: src/finance/expense-reimbursements.md (hunk 2 above)
Comment (monicazng) [reply src/finance/expense-reimbursements.md:55]: Sounds much better, thanks!

**Pull request merged**